src/python/grpcio/tests/unit/beta/test_utilities.py

```diff
@@ -29,10 +29,32 @@
 
 """Test-appropriate entry points into the gRPC Python Beta API."""
 
+import time
+from oauth2client import client as oauth2client_client
+from oauth2client import crypt
+
 from grpc._adapter import _intermediary_low
 from grpc.beta import implementations
 
+# TODO(kpayson) we should just add self_sign_jwt() to the 
+#ServiceAccountCredentials in the oauth2 library, this is a placeholder
+class JWTCredentials:
```

Comment:
Functionality-wise, this seems to work fine. But perhaps we could model this as a wrapper around googleCredentials that would give you a JWT token when `get_access_token().access_token` is invoked (that seems to be the way things will work once we add the functionality to oauth2 library).

If you do that, the resulting code would be very similar to what we'll have once we update oauth2 library.  (As you would be using the same factory method to create callCredentials from googleCredentials for all the use cases.)